client/components/admin/admin-pages-edit.vue

```diff
@@ -212,6 +247,77 @@ export default {
         message: `Coming soon...`,
         icon: 'directions_boat'
       })
+    },
+    /**
+     * Edit locale
+     */
+    editLocale () {
+      this.$store.commit('showNotification', {
+        style: 'indigo',
+        message: `Coming soon...`,
+        icon: 'directions_boat'
+      })
+    },
+    /**
+     * Focus an input after delay
+     */
+    focusField (ipt) {
+      this.$nextTick(() => {
+        _.delay(() => {
+          this.$refs[ipt].focus()
+        }, 200)
+      })
+    },
+    /**
+     * Update the current page
+     */
+    async updatePage() {
+      this.$store.commit(`loadingStart`, 'admin-users-update')
```

Comment:
Make sure all the loading triggers have a proper ID.

Reply:
True, will be fixed :)